lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,10 +1,26 @@
 module Pronto
   module Formatter
     class TextFormatter
+      include Colorizable
+
+      LOCATION_COLOR = :cyan
+
+      LEVEL_COLORS = {
+        info: :yellow,
+        warning: :magenta,
+        error: :red,
+        fatal: :red
+      }.freeze
+
+      attr_reader :config
+
+      def initialize(config = Config.new)
```

Comment:
I'm following the flow described by http://guides.rubygems.org/ in [Make your own gem](http://guides.rubygems.org/make-your-own-gem/).

First build/create the gem:
```bash
pronto$ gem build pronto.gemspec && gem install pronto-0.6.0.gem;
  Successfully built RubyGem
  Name: pronto
  Version: 0.6.0
  File: pronto-0.6.0.gem
Successfully installed pronto-0.6.0
1 gem installed
```

Run it:
```bash
pronto$ ruby -Ilib ./bin/pronto run
```

Enable it via `.pronto.yml` with [valid YAML bool](http://yaml.org/type/bool.html)
```yml
text_formatter:
  colors: on
```

Enable it ENV variable (`true` or anything other):
```bash
pronto$ TEXT_FORMATTER_COLORS=true ruby -Ilib ./bin/pronto run
```